app/javascript/styles/mastodon/accounts.scss

```diff
@@ -346,6 +346,10 @@
       color: $primary-text-color;
       font-weight: 700;
     }
+
+    .warning-hint {
```

Comment:
I'm not 100% sold on this, but given all hashtags are unreviewed by default, the "pending review" became very overbearing in the list view with font-weight 500.